app/src/main/java/fr/free/nrw/commons/depictions/DepictionDetailsActivity.java

```diff
@@ -0,0 +1,200 @@
+package fr.free.nrw.commons.depictions;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.FrameLayout;
+
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.viewpager.widget.ViewPager;
+
+import com.google.android.material.tabs.TabLayout;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import fr.free.nrw.commons.Media;
+import fr.free.nrw.commons.R;
+import fr.free.nrw.commons.depictions.Media.DepictedImagesFragment;
+import fr.free.nrw.commons.depictions.SubClass.SubDepictionListFragment;
+import fr.free.nrw.commons.explore.ViewPagerAdapter;
+import fr.free.nrw.commons.media.MediaDetailPagerFragment;
+import fr.free.nrw.commons.theme.NavigationBaseActivity;
+import fr.free.nrw.commons.upload.structure.depictions.DepictedItem;
+
+/**
+ * Activity to show depiction media, parent classes and child classes of depicted items in Explore
+ */
+public class DepictionDetailsActivity extends NavigationBaseActivity implements MediaDetailPagerFragment.MediaDetailProvider, AdapterView.OnItemClickListener {
```

Comment:
Please rename this class `WikidataItemDetailsActivity` (and same as appropriate for related variables/classes of that activity).

Reply:
done